tools/bin/capture-docker-stack

```diff
@@ -12,7 +12,7 @@ STACK_VERSION=$(echo "${STACK}" | cut -d '-' -f 2-)
 DOCKER_IMAGE=heroku/$STACK_NAME:$STACK_VERSION
 DOCKER_IMAGE_VERSION=$(docker inspect "${DOCKER_IMAGE}" | jq .[].Id | cut -d ':' -f 2 | cut -b 1-12)
 
-GIT_REF=${CIRCLE_TAG:-${CIRCLE_SHA1:?'Error: Either CIRCLE_TAG or CIRCLE_SHA1 must be set!'}}
+[[ ${GITHUB_REF_TYPE} == 'tag' ]] && GIT_REF=${GITHUB_REF_NAME:?'Error: GITHUB_REF_NAME must be set!'} || GIT_REF=${GITHUB_SHA:?'Error: either GITHUB_REF_NAME or GITHUB_SHA must be set!'}
```

Comment:
What do you think about using a multi-line conditional instead, to make it more readable?

Also, the SHA mode was something I added as part of adding TPS support, but we have never used it (and should probably not do an untagged release), so I think we could (/should) just remove the SHA support?

With that, perhaps this then becomes something like this?

```bash
if [[ ${GITHUB_REF_TYPE} == 'tag' ]]; then
    GIT_REF=${GITHUB_REF_NAME:?'Error: GITHUB_REF_NAME must be set!'}
else
    abort "Error: Attempting to publish to Heroku without having Git tagged!"
fi
```

Reply:
As an added bonus this protects against the GitHub Actions workflow config having a broken `if` conditional, and running publish by mistake.